README.md

```diff
@@ -135,42 +129,74 @@ The result will look like so:
 
 Now you have quite some info on the project, including its address.
 
-Other queries:
+## All queries:
+
+Fetch the batches of a user.
 
+```javascript
+fetchUserBatches(address: string);
 ```
-fetchUserBatches(address)
-fetchTCO2TokenById(tokenId)
-fetchAllTCO2Tokens()
-fetchUserRetirements()
+
+Fetch TCO2 token by it's address to get details like name, symbol, address and projectVintage.
+
+```javascript
+fetchTCO2TokenById(address: string);
 ```
 
-## Custom queries
+Fetch TCO2Token with the symbol to get details like name, symbol, address and projectVintage. A symbol would look like this: `TCO2-VCS-<projectId>-YYYY`. Make sure to input the full symbol.
 
-There's a lot more other pre-built subgraph queries that I could show you, but what I really want to show you is the `fetchCustomQuery` method.
+```javascript
+fetchTCO2TokenByFullSymbol(symbol: string);
+```
 
-This allows you to fetch with your own queries and can be very powerful if you know graphQL.
+Fetch all TCO2s details like name, symbol, address and projectVintage.
 
-- Getting your redeemed Tokens (TCO2s) you hold - useful for retirement of carbon credits.
+```javascript
+fetchAllTCO2Tokens();
+```
 
-```typescript
-import { gql } from "@urql/core";
+Fetch all user retirements to get information like creationTX, amount, token details and certificate details.
 
-const query = gql`
-  query ($id: String) {
-    user(id: $id) {
-      redeemsCreated {
-        token {
-          address
-          name
-        }
-      }
-    }
-  }
-`;
+```javascript
+fetchUserRetirements(address: string, first?: number, skip?: number);
+```
 
-const result = await toucan.fetchCustomQuery(query, { id: "1" });
+Fetch all redeems of a given pool to get information like amount, creator and token details. Only accepts the Pooltypes "NCT" or "BCT".
+
+```javascript
+fetchRedeems(poolSymbol: Pooltypes, first?: number, skip?: number);
+```
+
+Fetch all redeems of a user by pool and get information like amount and token details.
+
+```javascript
+fetchUserRedeems(address: string, poolSymbol: Pooltypes, first?: number, skip?: number);
+```
+
+Fetch TCO2 tokens that are part of the given pool and get information on name, amount, methodology and more.
+
+```javascript
+fetchPoolContents(poolSymbol: Pooltypes, first?: number, skip?: number);
+```
+
+Fetch a project by it's id to get more info on region and the standard.
+
+```javascript
+fetchProjectById();
 ```
 
+Fetch all aggregations (including for example, tco2TotalRetired or totalCarbonBridged).
+
+```javascript
+fetchAggregations();
+```
+
+## Custom queries
+
+There's a lot more other pre-built subgraph queries that I could show you, but what I really want to show you is the `fetchCustomQuery` method.
```

Comment:
Let's mention the example queries at https://thegraph.com/hosted-service/subgraph/toucanprotocol/matic